BatchDeciderTest fails in CI on the first pass

## Describe the bug
A [recently added test](https://github.com/CDCgov/prime-reportstream/blob/67229ec4858ca22c6bd9d3bfaa03e3115a32cf15/prime-router/src/test/kotlin/azure/batch/BatchDeciderTests.kt#L186-L223) in BatchDeciderTests.kt fails on the first try for every GitHub router build. 

## Impact on ReportStream
This test causes the CI to have to rerun tests on every build and thus consumes unnecessary resources and impacts engineer time when working on GitHub PRs.

## Steps to reproduce
Our router build is set to try a second time for any tests that fail. The see this in the logs they will need to be expanded to view all of the logs. The failure should be present on all router builds in GitHub:
```
[0K[1mgov.cdc.prime.router.azure.batch.BatchDeciderTests[22m run should enqueue correct number of batch messages with appropriate delays()[31m FAILED[31m

  java.lang.AssertionError: Verification failed: fewer calls happened than demanded by order verification sequence. 
  
  Matchers: 
  QueueAccess(#1322).sendMessageToQueue(any(), eq(covid-batch-queue), eq(PT0S)))
  QueueAccess(#1322).sendMessageToQueue(any(), eq(covid-batch-queue), eq(PT2M)))
  QueueAccess(#1322).sendMessageToQueue(any(), eq(covid-batch-queue), eq(PT4M)))
  
  Calls:
  
  
  Stack traces:
      at gov.cdc.prime.router.azure.batch.BatchDeciderTests.run should enqueue correct number of batch messages with appropriate delays(BatchDeciderTests.kt:211)
```

## Expected behavior
Unit tests should pass on the first run.

# Acceptance Criteria
 - [] The test is modified, or temporarily disabled and a new ticket logged if a larger fix is required.

--- a/prime-router/src/main/kotlin/azure/batch/BatchDeciderFunction.kt
+++ b/prime-router/src/main/kotlin/azure/batch/BatchDeciderFunction.kt
@@ -34,7 +34,7 @@ class BatchDeciderFunction(private val workflowEngine: WorkflowEngine = Workflow
         context: ExecutionContext?,
     ) {
         logger.trace("$batchDecider: Starting")
-//        try {
+        try {
             workflowEngine.db.transact { txn ->
                 // TODO: if for some reason the batch decider misses proper calculation of a receiver's batch run
                 //  we would want to pull all receivers with BATCH records that have next_action_at in the past and
@@ -64,10 +64,10 @@ class BatchDeciderFunction(private val workflowEngine: WorkflowEngine = Workflow
             }
 
             logger.trace("$batchDecider: Ending")
-//        } catch (e: Exception) {
-//            // Catching all exceptions, so Azure does not auto-implement the queue retry
-//            logger.error("$batchDecider function exception", e)
-//        }
+        } catch (e: Exception) {
+            // Catching all exceptions, so Azure does not auto-implement the queue retry
+            logger.error("$batchDecider function exception", e)
+        }
     }
 
     /**

--- a/prime-router/src/test/kotlin/azure/batch/BatchDeciderTests.kt
+++ b/prime-router/src/test/kotlin/azure/batch/BatchDeciderTests.kt
@@ -25,6 +25,7 @@ import org.jooq.tools.jdbc.MockResult
 import org.junit.jupiter.api.BeforeEach
 import java.time.Duration
 import java.time.OffsetDateTime
+import kotlin.test.Ignore
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
@@ -188,6 +189,7 @@ class BatchDeciderTests {
 
     // This test fails the first time it is run in the GitHub build on every single build. Skipping for now.
     @Test
+    @Ignore
     fun `run should enqueue correct number of batch messages with appropriate delays`() {
         // ensure this receiver is considered valid and due to batch
         every { timing1.isValid() } returns true

